crates/biome_js_analyze/src/lint/suspicious/no_confusing_labels.rs

```diff
@@ -56,6 +58,24 @@ declare_lint_rule! {
     /// $: { /* reactive block */ }
     /// </script>
     /// ```
+    /// ## Options
+    ///
+    /// Use the options to specify specific labels that you want to allow.
+    /// Labels can be used to mark code that should be removed in certain conditions,
+    /// for example in production builds.
```

Comment:
The phrase misses important context, that these labels are a esbuild/vite things. Can you please rephrase it, and maybe add a link to the feature?

Reply:
I have updated the documentation and linked to https://esbuild.github.io/api/#drop-labels. Is the new version acceptable?

Reply:
Yes, thank you!